### ISSUE TITLE
Font-Line height support

### ISSUE THREAD

**Issue by rolandpeelen** [blinksh/blink#675]
Hi Guys,

Might be related to: https://github.com/blinksh/blink/issues/643

I always have my fonts at a 1.5 or even 1.8 line height for better readabillity. Is this a possibility? Perhaps just having a place where you can enter some css just for this stuff?

Would be lovely!

**Comment by comfortablynick** [comment]
I've been trying to use the Iosevka font with ligatures and am running into what seems like a line height issue. Maybe the CSS idea would help; Iosevka looks great in hterm on Chrome and I'd love to use it on my iPad.

Might also be related to #668

**Comment by yury** [comment]
Hi @comfortablynick,

Are you trying to use pure losevka font or bundled by NerdFonts?

**Comment by comfortablynick** [comment]
Hi @yury,

I am using the pure Iosevka font. It's my first time trying to use a custom font, so I used a website that encodes the font files into the CSS file. I got it to load, but it looks exactly like the screen in #668 when there is a background color (e.g., in vim).

**Comment by yury** [comment]
I will check that.

What version do you use? default or terminal? A lot of versions :) need time to investigate.
https://github.com/be5invis/Iosevka/releases

Didn't know about this beautiful font. Will try to use it myself. Thank you!

**Comment by comfortablynick** [comment]
Yes there are a lot of versions! I use the default version, which has ligatures. It's a great font, my favorite so far and has powerline symbols built in.

**Comment by yury** [comment]
![AB7F8DE6-A84C-4F86-8D15-3BB85B5460B8](https://user-images.githubusercontent.com/5250/54479892-f35c7c00-4832-11e9-857b-895bd5fff66d.png)

**Comment by comfortablynick** [comment]
Sweet @yury! Looks great!

Please push out a beta when you get a chance. 😉
